src/main/java/com/epam/test_generator/services/EmailServiceProd.java

```diff
@@ -0,0 +1,91 @@
+package com.epam.test_generator.services;
+
+import com.epam.test_generator.controllers.user.UserDTOsTransformer;
+import com.epam.test_generator.controllers.user.response.UserDTO;
+import com.epam.test_generator.entities.Token;
+import com.epam.test_generator.entities.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletRequest;
+
+@Transactional
+@PropertySource("classpath:email.messages.properties")
+@Profile("!integration-tests")
+public class EmailServiceProd implements EmailService {
```

Comment:
Please think about better naming for this class

Reply:
Fixed